web/vite.config.cts

```diff
@@ -41,6 +41,50 @@ const sentryPluginOptions: SentryVitePluginOptions = {
   },
 };
 
+const PWAManifest: Partial<ManifestOptions> = {
```

Comment:
Would be nice if we could add these to support better installs :)
![Screenshot 2024-06-19 at 13 39 03](https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/d6d575b6-1a9e-4392-86c8-c2994252bb6e)

Reply:
They are added as a separate milestone in linear, to keep the effort and scope of the initial PR as small as possible.

Reply:
Great!